src/utils/constants.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class Status(Enum):
+    EXIT_SUCCESS = 0
+    EXIT_FAILURE = 1
+
+    INFERENCE_FAILURE = 2
+    CPP_BENCHMARK_FAILURE = 3
+    EXECUTOR_NOT_FOUND = 438
+    PROCESS_CREATE_ERROR = 127
```

Comment:
What about when inference is stopped by timeout?

Reply:
Added more return codes (including timeout)